boards/friendlyElec-nanoPiM1/default.nix

```
@@ -0,0 +1,19 @@
+{ lib, ... }:
+
+{
+  device = {
+    manufacturer = "FriendlyELEC";
+    name = lib.mkDefault "NanoPi M1";
+    identifier = lib.mkDefault "friendlyElec-nanoPiM1";
```

Using `lib.mkDefault` here would only be useful if you intend to have other boards `import` this as a base device.

For reference:

 - https://github.com/Tow-Boot/Tow-Boot/blob/79acaaba7f44f91c0e0e32cf9062f2cb443575e6/boards/radxa-RockPi4/default.nix#L6-L7
 - https://github.com/Tow-Boot/Tow-Boot/blob/79acaaba7f44f91c0e0e32cf9062f2cb443575e6/boards/radxa-RockPi4C/default.nix#L2-L4

And it only makes sense for boards that are a follow-up of a given one.

If it is planned for the future, you can add `lib.mkDefault` when contributing the additional boards.

```suggestion
    name = "NanoPi M1";
    identifier = "friendlyElec-nanoPiM1";
```